### PR TITLE
PR to update server README

### PR DESCRIPTION
**Pull request by BaderSerhan**

### CODE REVIEW

File: server/README.md [hunk 1]
```diff
@@ -3,17 +3,15 @@
...
-2. Ensure you have a <code>$GOPATH</code> environment variable set to a 
-directory of your choice. 
+2. [Ensure](https://golang.org/doc/code.html#GOPATH) you have a <code>$GOPATH</code> environment variable set to a directory of your choice. 
 
 #### MySQL Proxy
 1. [Install](https://cloud.google.com/sql/docs/mysql/quickstart-proxy-test#install-proxy) MySQL Proxy 
 2. Get your credentials file for your account from IAM (make sure it has 
 CloudSQL 
 Client, Editor or Admin permission)
-3. Run the proxy <code>cloud_sql_proxy -instances=wildfire-app-backend:asia-south1:wildfire-db1=tcp:3306  -credential_
```
Comment (BaderSerhan) [inline server/README.md]: I had to add `./` before `cloud_sql_proxy` for the command to run.

File: server/README.md [hunk 2]
```diff
@@ -3,17 +3,15 @@
 ## Requirements
 #### Go
 1. Install Go <code>brew install golang</code>
-2. Ensure you have a <code>$GOPATH</code> environment variable set to a 
-directory of your choice. 
+2. [Ensure](https://golang.org/doc/code.html#GOPATH) you have a <code>$GOPATH</code> environment variable set to a directory of your choice. 
 
 #### MySQL Proxy
 1. [Install](https://cloud.google.com/sql/docs/mysql/quickstart-proxy-test#install-proxy) MySQL Proxy 
 2. Get your credentials file for your account from IAM (make sure it has
```
Comment (BaderSerhan) [inline server/README.md]: Based on [this](https://cloud.google.com/docs/authentication/production) (the manual approach): I went to:

https://console.cloud.google.com/apis/credentials/
> Create Credentials > Service account key > Select new_service_account > Key type json > Create.

Is this the correct approach to get my credentials file? If yes, can you please add it to the README for the future?

File: server/README.md [hunk 3]
```diff
@@ -3,17 +3,15 @@
...
 2. Get your credentials file for your account from IAM (make sure it has 
 CloudSQL 
 Client, Editor or Admin permission)
-3. Run the proxy <code>cloud_sql_proxy -instances=wildfire-app-backend:asia-south1:wildfire-db1=tcp:3306  -credential_
-                       file=\<path to credential file\></code>
+3. Run the proxy <code>cloud_sql_proxy -instances=wildfire-app-backend:asia-south1:wildfire-db1=tcp:3306  -credential_file=\<path to credential file\></code>
  
 
 ## Runing the server
-<code>GO111MODULE=on go run server.go</code>
```
Comment (BaderSerhan) [inline server/README.md]: This command is not running for me. It is giving me this error `stat server.go file not found`.
I tried to have the project cloned locally and to run the command from there, but then I get another error: `default proxy initialization failed; consider calling proxy.Init explicitly: google: could not find default credentials`.

Comment (jamil-wyld) [reply server/README.md]: you need to run this in the server directory. Sorry I should make that clear. Because the README is in the server directory.

**Pull request merged**